insights/parsers/installed_product_ids.py

```diff
@@ -88,23 +88,37 @@ class InstalledProductIDs(CommandParser):
             Brand Type:
             Brand Name:
 
-    Filters have been added to the parser so that only the ``ID`` element will
-    be collected.
+    Filters have been added to the parser so that only the filtered lines will be collected.
 
     Attributes:
         ids (set): set of strings of the unique IDs collected by the command
+        product_certs(list): list of dicts of the product certificates key-value pairs
 
     Examples:
         >>> type(products)
         <class 'insights.parsers.installed_product_ids.InstalledProductIDs'>
         >>> list(products.ids)
         ['69']
+        >>> products.product_certs[0]
+        {'Path': '/etc/pki/product-default/69.pem', 'ID': '69'}
 
     """
     def parse_content(self, content):
         """ Parse command output """
         self.ids = set()
+        self.product_certs = []
+        one_file_data = None
         for line in content:
-            if line.strip().startswith('ID:'):
-                _, id = line.strip().split(':', 1)
-                self.ids.add(id.strip())
+            # different file delimiter
+            if line == 'Product Certificate':
+                if one_file_data:
+                    self.product_certs.append(one_file_data)
+                one_file_data = {}
+            else:
+                if one_file_data is not None:
+                    name, value = line.split(':', 1)
+                    one_file_data[name.strip()] = value.strip()
```

Comment:
For the key variables naming, to keep a better consistency, what about converting the key to the lowercase and joined the words with low dash ? 
 
For example, put the key variables name to: 
```
    assert results.product_certs[1]['Path'] == '/etc/pki/product/479.pem'.    ----> 'path'
    assert results.product_certs[1]['ID'] == '479'                            ----> 'id'
    assert results.product_certs[1]['Brand Type'] == ''                       ----> 'brand_type'
```

Reply:
yes, good to me.